SPECS-SIGNED/systemd-boot-signed/systemd-boot-signed.spec

```diff
@@ -0,0 +1,91 @@
+%global debug_package %{nil}
+%ifarch x86_64
+%global buildarch x86_64
+%endif
+
+# Support for quick builds with rpmbuild --build-in-place.
+# See README.build-in-place
+%bcond inplace 0
+Summary:        Signed systemd-boot for %{buildarch} systems
+Name:           systemd-boot-signed-%{buildarch}
+%if %{without inplace}
+Version:        255
+%else
+# determine the build information from local checkout
+Version:        %(tools/meson-vcs-tag.sh . error | sed -r 's/-([0-9])/.^\1/; s/-g/_g/')
+%endif
+Release:        14%{?dist}
+License:        LGPL-2.1-or-later AND MIT AND GPL-2.0-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://systemd.io
+# This package's "version" and "release" must reflect the unsigned version that
+# was signed.
+# An important consequence is that when making a change to this package, the
+# unsigned version/release must be increased to keep the two versions consistent.
+# Ideally though, this spec will not change much or at all, so the version will
+# just track the unsigned package's version/release.
+#
+# To populate these sources:
+#   1. Build the unsigned packages as normal
+#   2. Sign the desired binary
+#   3. Place the unsigned package and signed binary in this spec's folder
+#   4. Build this spec
+Source0:        systemd-boot-unsigned-%{version}-%{release}.%{buildarch}.rpm
+Source1:        systemd-bootx64.efi
+ExclusiveArch:  x86_64
+
+%description
+This package contains the systemd-boot EFI binary signed for secure boot. The package is
+specifically created for installing on %{buildarch} systems
+
+%package -n     systemd-boot-signed
+Summary:        UEFI boot manager (signed version)
+
+Provides: systemd-boot-signed-%{efi_arch} = %version-%release
+Provides: systemd-boot = %version-%release
+Provides: systemd-boot%{_isa} = %{version}-%{release}
+# A provides with just the version, no release or dist, used to build systemd-boot
+Provides: version(systemd-boot-signed) = %version
+Provides: version(systemd-boot-signed)%{_isa} = %version
+
+# self-obsoletes to install both packages after split of systemd-boot
+Obsoletes: systemd-udev < 252.2^
+
+%description -n systemd-boot-signed
+systemd-boot (short: sd-boot) is a simple UEFI boot manager. It provides a
+graphical menu to select the entry to boot and an editor for the kernel command
+line. systemd-boot supports systems with UEFI firmware only.
+
+This package contains the signed version that works with Secure Boot.
+
+%prep
+
+%build
+mkdir rpm_contents
+pushd rpm_contents
+
+# This spec's whole purpose is to inject the signed systemd-boot binary
+rpm2cpio %{SOURCE0} | cpio -idmv
+cp %{SOURCE1} ./usr/lib/systemd/boot/efi/systemd-bootx64.efi
+
+popd
+
+%install
+pushd rpm_contents
+
+# Don't use * wildcard. It does not copy over hidden files in the root folder...
+cp -rp ./. %{buildroot}/
+
+popd
+
+%files -n systemd-boot-signed
```

Comment:
Double-checking the name. So far the "signed" specs generated RPMs without the words "signed" or "unsigned" in them. That is because the "unsigned" versions were not the packages we end up publishing, so the "signed" ones had to provide the correctly-named packages. Is there a reason we're doing this one differently?

Reply:
ah.. No particular reason. The unsigned package was explicitly named `systemd-boot-unsigned` so I thought it wouldn't make sense to override that name for the signed version. I guess to keep it consistent with other packages, I'll have to remove the "signed/unsigned" bit from both versions of systemd-boot

Reply:
Renamed systemd-boot-unsigned to systemd-boot. Updated the signed spec and pipeline PR